frontend/pages/club/[club]/index.tsx

```diff
@@ -91,6 +91,15 @@ const ClubPage = ({
     setClub(newClub)
   }
 
+  useEffect(() => {
+    doApiRequest('/clubvisits/?format=json', {
+      method: 'POST',
+      body: {
+        club: code,
+      },
+    })
+  })
```

Comment:
@cphalen This will send club visit requests to the server in an infinite loop, since you did not specify the second argument of `useEffect`. Please modify this to have `[]` as the second argument.